pkg/tools/otelcol-config/opamp.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+
+	"github.com/mikefarah/yq/v4/pkg/yqlib"
+)
+
+var errRemoteControlNotEnabled = errors.New("remote control not enabled")
+
+func SetOpAmpEndpointAction(ctx *actionContext) error {
+	endpoint := ctx.Flags.SetOpAmpEndpoint
+	if err := validateEndpoint(endpoint); err != nil {
+		return err
+	}
+	conf, err := ReadConfigDir(ctx.ConfigDir)
+	if err != nil {
+		return err
+	}
+
+	writer := ctx.WriteSumologicRemote
+	doc := conf.SumologicRemote
```

Comment:
Nitpick: not 100% sure about the name of `doc`

Reply:
It's a yaml doc